std/math.d

```diff
@@ -3927,6 +3927,91 @@ float floor(float x) @trusted pure nothrow @nogc
     assert(isNaN(floor(float.init)));
 }
 
+/**
+ * Round `val` to a multiple of `unit`. `rfunc` specifies the rounding
+ * function to use; by default this is `rint`, which uses the current
```

Comment:
`round()`, `lround()`, `rint()`, `lrint()`, `nearbyint()`, and `rndtol()` all do almost the same thing, and should (in my opinion) be consolidated into a single `round()` function with some template parameters.

This would obviously require a long deprecation cycle, though.